samples/model-builder/conftest.py

```diff
@@ -364,3 +364,48 @@ def mock_endpoint_explain(mock_endpoint):
     with patch.object(mock_endpoint, "explain") as mock_endpoint_explain:
         mock_get_endpoint.return_value = mock_endpoint
         yield mock_endpoint_explain
+
+
+"""
+----------------------------------------------------------------------------
+FeatureStore Fixtures
+----------------------------------------------------------------------------
+"""
+
+
+@pytest.fixture
+def mock_featurestore():
+    mock = MagicMock(aiplatform.Featurestore)
+    yield mock
+
+
+@pytest.fixture
+def mock_init_featurestore(mock_featurestore):
+    with patch.object(aiplatform, "Featurestore") as mock:
+        mock.return_value = mock_featurestore
+        yield mock
+
+
+@pytest.fixture
+def mock_create_featurestore():
+    with patch.object(aiplatform.Featurestore, "create") as mock:
+        yield mock
+
+
+@pytest.fixture
+def mock_create_entity_type():
+    with patch.object(aiplatform.EntityType, "create") as mock:
+        yield mock
+
```

Comment:
you will need `mock_entity_type` and `mock_feature` similar to `mock_featurestore`, which are the return value for `mock_create_*` and `mock_get_*`

Reply:
Added functions